From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Thu, 18 Apr 2019 11:07:51 -0700
Subject: Revert "Roll clang 356356:357569."

This reverts commit a50f2f33843d26fd7c0d1f1a2331aa45a392c6cd.

This roll broke ARM builds. Tracking bug: crbug.com/953815

diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index ab0311650bffadf8d3cfe03f8a55cea319e15550..b9faaefcd3c5f978af9d72f08f1ba230d5a27b24 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -35,10 +35,10 @@ import zipfile
 # Do NOT CHANGE this if you don't know what you're doing -- see
 # https://chromium.googlesource.com/chromium/src/+/master/docs/updating_clang.md
 # Reverting problematic clang rolls is safe, though.
-CLANG_REVISION = '357692'
+CLANG_REVISION = '356356'
 
 # This is incremented when pushing a new build of Clang at the same revision.
-CLANG_SUB_REVISION=1
+CLANG_SUB_REVISION=3
 
 PACKAGE_VERSION = "%s-%s" % (CLANG_REVISION, CLANG_SUB_REVISION)
 
